helm/adelphi/values.yaml

```diff
@@ -1,8 +1,8 @@
 namespace: cass-operator
-clusterSize: 1 # number of nodes per cluster
-allowMultipleNodesPerWorker: true
+clusterSize: 3 # number of nodes per cluster
+allowMultipleNodesPerWorker: false
 
-storageClassName: local-path
+storageClassName: standard
```

Comment:
the default storage name in GKE is "standard", and for k3d it is "local-path"

I opened https://github.com/datastax/adelphi/issues/36 in order to have different sample config files per environment